.github/workflows/release-nightly.yml

```diff
@@ -19,20 +19,37 @@ jobs:
       build: ${{ matrix.build }}
 
   release:
-    name: Make Release on GitHub
+    name: Make Release on GitHub and pypi.anaconda.org
```

Comment:
Should we make this into 2 separate jobs? One to release on github, the other on pypi?
Would make it clearer what steps belong to what, and also easier to remove the github upload in the future if we want to do that?
I guess both jobs could run in parallel?